Menubar component is not working

**I'm submitting a ...**  (check one with "x")
```
[X ] bug report
[ ] feature request
[ ] support request => Please do not submit support request here, instead see https://forum.primefaces.org/viewforum.php?f=57
```
Menubar component is not working. Looks to be an issue with camelcase references. It's looking for a file called Menu**B**ar.js _(Capital B)_ rather than Menu**b**ar.js as it's saved in node modules. 

> ./node_modules/primereact/menubar.js
> Module not found: `/Users/jordan/Development/Projects/parchment/node_modules/primereact/components/menubar/MenuBar.js` does not match the corresponding path on disk `Menubar.js`.

--- a/exports/menubar.d.ts
+++ b/exports/menubar.d.ts
@@ -1,1 +1,1 @@
-export * from './components/menubar/MenuBar';
+export * from './components/menubar/Menubar';

--- a/exports/menubar.js
+++ b/exports/menubar.js
@@ -1,3 +1,3 @@
 'use strict';
 
-module.exports = require('./components/menubar/MenuBar');
+module.exports = require('./components/menubar/Menubar');

